src/content/docs/mlops/bring-your-own/mlops-byo.mdx

```diff
@@ -5,33 +5,66 @@ redirects:
+
+
+## Easy to Integrate [#integrate]
+The [ml-performance-monitoring](https://github.com/newrelic-experimental/ml-performance-monitoring) python package, based on [newrelic-telemetry-sdk-python](https://github.com/newrelic/newrelic-telemetry-sdk-python)​​, allows you to send your model’s features and prediction values, as well as custom metrics, by simply adding a few lines to your code.
+
+## View Model Predictions [#view-predictions]
+By monitoring the performance of your machine learning models, you can identify any issues that may be impacting their accuracy or performance. You’ll see vital information in your dashboard like your total model predictions and distribution over time. Additionally, you can set alerts to track any changes in total predictions or high/low picks.
+
+<img
+  title="An image of the model performance and predictions in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops1}
+/>
+ 
+## Detect Model and Data Drift [#detect-drift]
+With real time monitoring, you can quickly identify issues with your machine learning models by detecting model and data drift. Model and data drift are changes in your real world environment that can affect the predictive power of your model. By getting real-time data on your model and data drift, you can quickly take corrective action before those issues impact the performance of your applications.
+
+<img
+  title="An image of the model and data drift in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops2}
+/>
+
+## Dive Deeper into Model Features [#view-model-features]
+Get insight and statistical data on your features and get alerted on any deviation from standard and expected behavior.
+
+<img
+  title="An image of the model features in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops3}
+/>
+
+<img
+  title="An image of the feature distribution in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops4}
+/>
 
-2. **Data metrics:** Instead of sending all your raw inference data, select the aggregated statistics option over the features, and predictions (for example, `min`, `max`, `average`, or `percentile`). They will be sent automatically as **[metrics](/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/)**.
+<img
+  title="An image of the model count, missing values, and standard deviation in New Relic"
+  alt="An image of the New Relic MLOps dashboard."
+  src={mlops5}
+/>
 
-3. **Custom metrics:** Calculate your own metrics, monitor your model performance or model data, and stream them to New Relic using the `record_metrics` function. They are sent as **[metrics](/docs/data-apis/ingest-apis/metric-api/introduction-metric-api/)**.
+## Want these Benefits? [#get-benefits]
```

Comment:
Lets change to "How to start monitoring your ML model performance?"

Reply:
done